networkx/algorithms/boundary.py

```diff
@@ -121,6 +135,20 @@ def node_boundary(G, nbunch1, nbunch2=None):
     set
         The node boundary of `nbunch1` with respect to `nbunch2`.
 
+    Examples
+    -------
+    >>> G = nx.gnp_random_graph(10, 0.3, seed = 100)
```

Comment:
Similar comment here -- is there a good graph that shows the node boundaries?

Reply:
Same change